src/en/space-station-14/round-flow/proposals/station-ai.md

```diff
@@ -0,0 +1,162 @@
+  - Backup power? Camera built in?
+- Crew devices; computers, machines, servers
+  - These don’t contribute as much **Compute** as they are not specialized for AI use. Servers do provide more **Compute** than just consoles, and provide **Bandwidth** in amounts comparable or identical to dedicated machines..
+  - Locks out the device for crew use, rendering it inoperable. This acts as a conflict point with the crew. Computers don’t work, machines either. Servers don’t provide normal functionality or do so in a very limited way.
+    - E.g. crew monitoring server could give invalid/partial data with a warning, or just not work at all. Same for telecom servers.
+  - Non-antag AI can be flushed out from the device via a near-instant crew exposed action. This includes a subverted AI. Could be an UI that shows “This device is in use by AI \<name\>. \<Override (button)\>.”
+  - Such shooing away should keep the AI from taking the device over for at least some time, at least a minute. It should not be able to just annoy the crew to death even when compromised - it’s still shackled.
+  - Antagonist AI, “Malf”, should have an option or be able to spend some resources to completely lock the crew out. Will require more detailed balancing later.
+  - For “Malf”, the timeout should be reduced or bypassable. Again, might need some balance tweaking whether it’s just a pop up “you sure?” or a resource sink.
+- Borgs! Useful and somewhat plentiful, perfect for an emergency ~~snack~~ method of survival when the station’s powered off and the AI SMES is about to drain, with a downside of rendering the borg unable to synthesize voice (speak) when AI’s along for the ride, except for beeps and emotes. Does not require borg sync, but requires approval from the borg even if synced. Malf AI can bypass the approval.
+
+The AI satellite has several of the AI machines built into it, providing the base necessary resources for the AI, and some extra to make it actually useful to the crew.
+
+**Note:** Something noted by other people is how this system would interact with multiple AI. Two AI might be possible with either significant crowding and cut down resources, or a very annoyed crew as two AI fight one another for scraps. They should be able to lock one another out from a device, and require significant investment of power to pull control away. Of course, the crew can always just shoo them away with same near-instant action.
+
+## Vision - Cameras and Map
+
+The AI is not a biological entity, and it’s only barely a “physical” one. It doesn’t need to care about specifics of how things look, but rather whether or not an area is intact, safe, and secure. As such, the vision system is an optional ability that an AI player can shut down, whether to free space up for something else, or as an emergency measure.
```

Comment:
Agree. This requires the NavMap to be fleshed out further (of course), which you describe below.

AI should be able to do a LOT, even without TV vision.